build-system/amp.extern.js

```diff
@@ -48,7 +48,7 @@ function ExtensionPayload() {}
 /** @type {string} */
 ExtensionPayload.prototype.n;
 
-/** @type {function(!Object)} */
+/** @type {function(!Object,!Object)} */
```

Comment:
same question, this is forced as mandatory even on the multi pass closure builds?